docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/insert-vertex-syntax.md

```diff
@@ -11,7 +11,7 @@ prop_value_list:
   [prop_value [, prop_value] ...]
 ```
 
-INSERT VERTEX statement inserts one vertex into **Nebula Graph**.
+INSERT VERTEX statement inserts vertex into **Nebula Graph**.
```

Comment:
"the INSERT VERTEX statement" 

"vertex" is a countable noun. use an article before it or use the plural form.